app/views/members/activities/partials/_view.html.haml

```diff
@@ -68,6 +68,12 @@
         %span.activity-unenroll
           = l(activity.unenroll_date, format: "%A %d/%m/%Y")
           23:59
+        <br/>
+      - if !activity.payment_deadline.nil?
+        %strong= I18n.t('activerecord.attributes.activity.payment_deadline')
+        %span.activity-payment-deadline
+          = l(activity.payment_deadline, format: "%A %d/%m/%Y")
+          23:59
```

Comment:
Do we want to display the payment deadline to the users? Or maybe only display it when it differs from the default?

Reply:
Yeah why not. The default is pay whenever the treasurer gets angry, but always after the activity. For the studytrip and airco this differs. I think only showing it when it is set (and thus when it is relevant) is the best option.

Reply:
Alright, makes sense